zadaci/sa-kolokvijuma/2020/RA/T34/G5/zadatak.rst

```diff
@@ -1,9 +1,9 @@
 Iz zadate ulazne datoteke učitatati statički niz struktura, pri čemu se struktura
-rodjendan_st sastoji od sledećih polja:
+rođendan_st sastoji od sledećih polja:
```

Comment:
This should have stayed "rodjendan_st", since it is a name of the struct in C code.

Reply:
Ah sorry, thank you, fixed :)